shell/common/animator.cc

```diff
@@ -25,7 +25,8 @@ Animator::Animator(Delegate& delegate,
     : delegate_(delegate),
       task_runners_(std::move(task_runners)),
       waiter_(std::move(waiter)),
-      last_begin_frame_time_(),
+      last_frame_begin_time_(),
```

Comment:
i've also renamed this from `last_begin_frame_time_` to `last_frame_begin_time_`, I think this reads better.